pkg/noderesourcetopology/cache/overreserve.go

```diff
@@ -101,17 +101,16 @@ func NewOverReserve(ctx context.Context, lh logr.Logger, cfg *apiconfig.NodeReso
 func (ov *OverReserve) GetCachedNRTCopy(ctx context.Context, nodeName string, pod *corev1.Pod) (*topologyv1alpha2.NodeResourceTopology, CachedNRTInfo) {
```

Comment:
this is a u/s commit right? if so why it has the `[KNI]` prefix? https://github.com/openshift-kni/scheduler-plugins/blob/master/RESYNC.md#patching-openshift-kni-specific-commits this is not a d/s specific change (e.g a change which u/s cannot accept or don't care about)

Reply:
totally, but is not so easy to implement so this is why it keeps getting deferred. I think you can pass the check by adding a top-level commit updating our `RESYNC.log.md`. That commit will have the `[KNI]` prefix so it can pass the checks (and is not wrong either). See: 780ea849857ee3504f363bf64549e702a925478d

Reply:
unfortunatly that didn't work, we have to either update each cherry-picked commit or skip the check completely like we do for resyncs:
https://github.com/openshift-kni/scheduler-plugins/pull/357

Reply:
ok, let's check what we did in the past